CONTRIBUTING.md

```diff
@@ -127,25 +129,47 @@ Excellent [PR guidelines](https://github.com/kubernetes/community/blob/master/co
 You will typically receive feedback from other developers. Before responding, take a moment to review the 
 [Code of Conduct](https://github.com/decred/dcrdata/blob/master/CODE_OF_CONDUCT.md). 
 
-Work through the comments and resolve any confusion with others and then implement the changes necessary.
+Work through the comments and resolve any confusion with others. Make whatever revisions are necessary.
 
 11. Resubmitting
 
-Similar to step #6 above, you will want to clean up your commits before creating the pull request. 
-As long as it hasn't been a very long time, you don't need to rebase to master. Instead, rebase 
-to the commit of the last pull request. To see a list of commits, you can use
+Commit your work as in step 5 above.
+
+###### a)
+
+Before resubmitting, clean up any little touchip commits you've made since the last time you pushed.
+If you've only made one commit since then, you can skip this step.
+For example, if you have made 3 commits since your last push, then run the following to put them together.
+
+```sh
+git rebase -i HEAD~3
+```
+
+The number after the tilda (~) is the number of commits that you want to combine, including the one you did at the beginning of this step.
+
+###### b)
+
+Then rebase the entire branch back to an updated master. 
 
 ```sh
-git log --oneline --graph
+git checkout master
+git pull https://github.com/decred/dcrdata.git

```

Comment:
I suppose this is fine, but it is different from my workflow so I can't say it's not going to have issues.  I just make sure my local `master` is tracking the remote master with the url https://github.com/decred/dcrdata (whatever origin `git remote -v` says tracks the official decred repo).  I've noticed some people have their local `master` tracking `theirremote/master`, which tends to cause confusion.  And there is no need to have a separate master in the the dev's own fork.  I don't want to force people to adopt a workflow that does not suit them, but I would suggest to advice having `master` tracking the remote for `https://github.com/decred/dcrdata` OR pull into and rebase on whatever local branch is tracking `https://github.com/decred/dcrdata`.